client/js/verification-portal/get-resource-to-verify-button.js

```diff
@@ -87,8 +87,10 @@ const populateForm = ({ changeFieldValue, resource }) => {
 		_.get(verifiedData, 'primaryContactPerson.verified'),
 	)
 
-	_.forEach(services, (value, key) => {
-		changeFieldValue(`services.${key}`, 1)
+	_.forEach(services, value => {
```

Comment:
What do you think about changing `value` to `service` so it's more clear what the data is?